src/main/java/ua/bookstore/online/mapper/UserMapper.java

```diff
@@ -12,5 +12,7 @@ public interface UserMapper {
     UserResponseDto toDto(User user);
 
     @Mapping(target = "id", ignore = true)
+    @Mapping(target = "roles", ignore = true)
+    @Mapping(target = "authorities", ignore = true)
```

Comment:
I don't see any fields "authorities" in you code. For what this field?

Reply:
Mapstruct look at name of methods in entity class, and it has getAuthoritis...

Reply:
Agree